tools/misc/tpm2_print.c

```diff
@@ -11,6 +11,9 @@
 #include "tpm2_convert.h"
 #include "tpm2_tool.h"
 #include "tpm2_util.h"
+#include "object.h"
+
+DECLARE_ASN1_FUNCTIONS(TSSPRIVKEY_OBJ);
```

Comment:
Does it make sense to move the fetch functions and the required structures and headers to tpm2_openssl.h including this macro for declaring the asn1 functions so if someone needs to compile out tpm2_print tool, the load and loadexternal could still work?